client/src/utils/algolia-locale-setup.ts

```diff
@@ -26,30 +26,30 @@ const algoliaIndices = {
     searchPage: 'https://www.freecodecamp.org/italian/news/search/'
   },
   portuguese: {
-    name: 'news-pt',
+    name: 'news-pt-br',
     searchPage: 'https://www.freecodecamp.org/portuguese/news/search/'
   },
-  // TODO: Replace with Ukrainian news when we have more useful resources on that instance
   ukrainian: {
-    name: 'news',
-    searchPage: 'https://www.freecodecamp.org/news/search/'
+    name: 'news-uk',
+    searchPage: 'https://www.freecodecamp.org/ukrainian/news/search/'
   },
   japanese: {
     name: 'news-ja',
     searchPage: 'https://www.freecodecamp.org/japanese/news/search/'
   },
-  // TODO: Replace with German news when we have more useful resources on that instance
+  korean: {
+    name: 'news-ko',
+    searchPage: 'https://www.freecodecamp.org/korean/news/search/'
+  },
+  // Note: We don't build News for the locales below, so show English
+  // hits and use the English search page
   german: {
     name: 'news',
     searchPage: 'https://www.freecodecamp.org/news/search/'
   },
   swahili: {
-    name: 'news-sw',
+    name: 'news',
     searchPage: 'https://www.freecodecamp.org/swahili/news/search/'
```

Comment:
```suggestion
    searchPage: 'https://www.freecodecamp.org/news/search/'
```
There's no swahili news at the moment.

Reply:
Thanks for catching that, @ojeytonwilliams. Totally missed that 🤦